src/Core/Settings.h

```diff
@@ -605,7 +605,7 @@ static constexpr UInt64 operator""_GiB(unsigned long long value)
     M(Bool, read_from_filesystem_cache_if_exists_otherwise_bypass_cache, false, "Allow to use the filesystem cache in passive mode - benefit from the existing cache entries, but don't put more entries into the cache. If you set this setting for heavy ad-hoc queries and leave it disabled for short real-time queries, this will allows to avoid cache threshing by too heavy queries and to improve the overall system efficiency.", 0) \
     M(Bool, enable_filesystem_cache_on_lower_level, true, "If read buffer supports caching inside threadpool, allow it to do it, otherwise cache outside ot threadpool. Do not use this setting, it is needed for testing", 0) \
     M(Bool, skip_download_if_exceeds_query_cache, true, "Skip download from remote filesystem if exceeds query cache size", 0) \
-    M(UInt64, max_query_cache_size, (128UL * 1024 * 1024 * 1024), "Max remote filesystem cache size that can be used by a single query", 0) \
+    M(UInt64, max_query_cache_size, 128_GiB, "Max remote filesystem cache size that can be used by a single query", 0) \
```

Comment:
We will have the "query cache" feature soon.
The name of this setting is ambiguous and it will be confused with the upcoming feature.

Please rename to "max_filesystem_cache_usage_per_query".

PS. "remote" - the filesystem cache is no longer limited to remote filesystems, let's remove this word.